README.md

```diff
@@ -81,6 +81,11 @@ Internet-accessible deployment the following variables will need to be exported:
 
 - `BODY_PARSER_LIMIT` - Increase the maximum payload size that our [body parser][body-parser] will attempt to parse. Useful for github web hooks.
 - `DEBUG` - Set this to `strider*` to enable all debug output. This is very helpful when troubleshooting issues or finding the cause of bugs in Strider. For more information see https://www.npmjs.com/package/debug
+- `jobsQuantityOnPage` (in rc file) - Object which defines configuration for jobs quantity to display on project page. It has the following fields:
```

Comment:
user will be very surprised to see smth about rc file

Reply:
2078aedb67b879d4c9934b989cc4a9e1a83123b9